src/Components/Product/ProductListing.css

```diff
@@ -0,0 +1,49 @@
+
+.product-view-container {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+}
+
+.filter-container {
+    min-width: 15.75rem;
+    max-width: 15.75rem;
+    height: 100%;
+    flex-grow: 0 !important;
```

Comment:
!important should be last resort, it would be better if you could implement same with some other alternative.

Reply:
Resolved this